webapp-django/crashstats/signature/jinja2/signature/signature_summary.html

```diff
@@ -1,4 +1,4 @@
-<div class="panel tab-inner-panel">
+<div class="panel tab-inner-panel" id="tab-platform">
```

Comment:
Can you replace ``tab`` with ``panel`` in all these ids? They aren't tabs but panels, and it might conflict with the actuals tabs of that page.